infrastructure/src/main/java/org/wso2/carbon/testgrid/infrastructure/providers/ShellScriptProvider.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.testgrid.infrastructure.providers;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.testgrid.common.*;
+import org.wso2.carbon.testgrid.common.exception.TestGridInfrastructureException;
+
+import java.nio.file.Paths;
+
+/**
+ * This class creates the infrastructure for running tests
+ */
+public class ShellScriptProvider implements InfrastructureProvider {
+    private static final Log log = LogFactory.getLog(ShellScriptProvider.class);
+    private final static String SHELL_SCRIPT_PROVIDER = "Infra Create";
```

Comment:
It's better to have a more meaningful name for SHELL_SCRIPT_PROVIDER.

Reply:
Fixed in commit 21bb7f6